src/pages/edge/lifecycle-for-edge-network/index.md

```diff
@@ -11,9 +11,23 @@ keywords:
 
 The Adobe Experience Platform Mobile SDK Lifecycle extension enables application lifecycle data collection from your mobile app when using the Adobe Experience Platform Mobile SDK and the [Edge Network extension](../edge-network/index.md). Use the following steps to set up an application to forward Lifecycle extension data to the Adobe Experience Platform.
 
-## Add AEP Mobile Lifecycle Details field group to your XDM Schema in Adobe Experience Platform
+## Migrating from Analytics to Lifecycle for Edge Network
 
-Include the "AEP Mobile Lifecycle Details" field group to the schema defined in your Edge configuration data stream. This field group defines all the data auto-collected by the Lifecycle for Edge Network extension.
+If you are an existing Mobile Analytics customer and are migrating to the Edge Network, or would like more information on how Lifecycle for Edge Network works, please read the [Lifecycle for Edge Network migration reference](./migration-reference.md). Here you will find important information about the differences of how Lifecycle works when using Edge Network versus Analytics.
+
+<InlineAlert variant="warning" slots="text"/>
+
+After migrating to Lifecycle for Edge Network from Analytics, the number of events (hits) sent to Adobe Experience Platform may increase and could impact billable events. It is important to understand the changes to Lifecycle for Edge Network by reading the [migration reference](./migration-reference.md).
+
+## Configure and install Lifecycle for Edge Network
+
+### Add AEP Mobile Lifecycle Details field group to your XDM Schema in Adobe Experience Platform
+
+Include the "AEP Mobile Lifecycle Details" field group to the schema defined in your Edge configuration datastream. This field group defines all the data auto-collected by the Lifecycle for Edge Network extension.
+
+<InlineAlert variant="info" slots="text"/>
+
+If your workflow does not require the creation of an XDM schema, such as when only using the Adobe Analytics service, then this step is optional.
```

Comment:
Should we add more context to make it more evident, something on these lines:
... such as when only using the Adobe Analytics service and no AEP services ...

Reply:
I don't want to get into a situation where we list all the services which require a schema as that can change in the future. It looks like, however, that only AEP and Data Prep require a schema. How about I rewrite the info block as

> Creating an XDM schema is optional if your workflow does not require one. An XDM schema is required for some services, such as Adobe Experience Platform and Data Prep for mapping XDM fields on the server. However, if the datastream only enables the Analytics service, for example, then an XDM schema is not strictly required.

Reply:
Rewrite looks good!